apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -620,6 +758,33 @@ export const FileBrowser = React.memo(() => {
     [project.folders, dialogControl, renameFolder, appName]
   );
 
+  const handleDragEnd = useMemo(
+    () => (e: DragOverEvent) => {
+      if (e?.over) {
+        if (e.active.data.current?.type === 'FOLDER') {
+          moveFolder(e.active.data.current.id as string, e.over.id as string);
+        } else if (e.active.data.current?.type === 'FILE') {
+          moveFile(e.active.data.current.id as string, e.over.id as string);
+        }
+      }
+    },
+    [moveFile, moveFolder]
+  );
+
+  const sensors = useSensors(
+    useSensor(PointerSensor, {
+      activationConstraint: {
+        distance: 2,
+      },
+    })
+  );
+
+  Object.values(project.folders).forEach(f => {
```

Comment:
what is the purpose of this?

Reply:
Nuked it, it was repair code. While I was testing it I had a bug that blew up the folder structure, so this just reset things back to a usable state. Not needed going forward.